modules/core/README.md

```diff
@@ -373,18 +373,18 @@ For more information, run
 ./upgrade.py -h
 ```
 
-#### Manual upgrading Consul
+#### Manually upgrading Consul
 
 **Important**: It is important that you only terminate Consul instances one by one. Make sure the
 new servers are healthy and have joined the cluster before continuing. If you lose more than a
 quorum of servers, you might have data loss and have to perform
 [outage recovery](https://www.consul.io/docs/guides/outage.html).
 
 1. Build your new AMI, and Terraform apply the new AMI.
-1. Terminate the instance that you would like to remove.
-1. The Consul server will gracefully exit, and cause the node to become unhealthy, and AWS will
+2. Terminate the instance that you would like to remove.
+3. The Consul server will gracefully exit, and cause the node to become unhealthy, and AWS will
    automatically start a new instance.
-1. Make sure the new instance started by AWS is healthy before continuing. For example, use
+4. Make sure the new instance started by AWS is healthy before continuing. For example, use
```

Comment:
Just to note, the original wasn't a mistake, because markdown automatically increment the bullet number by 1 (you can always try previewing in GitHub).

Nevertheless, I do prefer incrementing manually.